tests/commands/update/test_update_case_cmd.py

```diff
@@ -229,5 +229,5 @@ def test_update_case_change_sv_vcf_research_path(mock_app, case_obj, sv_clinical
     assert sum(1 for i in store.variant_collection.find({"category": "sv"})) > 0
     res = store.variant_collection.find({"category": "sv", "variant_rank": {"$gt": 10}})
     assert sum(1 for i in res) == 0
-    res = store.case_collection.find({"sv_rank_model_version": 1.5})
+    res = store.case_collection.find({"sv_rank_model_version": "1.5"})
```

Comment:
This is a string according to the Case model: https://github.com/Clinical-Genomics/scout/blob/f73d376002e8d29b1133f6c60e55694958763ebd/scout/models/case/case.py#L87

Why didn't this test fail before I wonder..